google-cloud-compat-checker/README.md

```diff
@@ -0,0 +1,25 @@
+Google Cloud Java Compatibility Checker
```

Comment:
What do you think about renaming this into something more generic (as I believe more utility stuff can/will be added here in the future)?